server/controllers/submission.js

```diff
@@ -0,0 +1,40 @@
+const asyncHandler = require('express-async-handler');
+const ObjectId = require('mongoose').Types.ObjectId;
+const Contest = require('../models/Contest');
+const Submission = require('../models/Submission');
+
+//@GET /:id
+//Get all submissions only for the contest owner
+exports.getAllSubmissionForOwner = asyncHandler((req, res, next) => {
```

Comment:
Can we not incorporate the logic for both use cases within this one controller? :)

Reply:
sorry, what a stupid mistake!!